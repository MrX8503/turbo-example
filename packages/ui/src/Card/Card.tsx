import type { ReactNode } from 'react';

export interface CardProps {
    children?: ReactNode;
}

export default function Card ({ children }: CardProps) {
    return (
        <>
            <h1>Card Component</h1>
            {children}
        </>
    );
}
