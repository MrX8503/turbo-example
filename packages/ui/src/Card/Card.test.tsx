import renderer from 'react-test-renderer';

import Card from './Card';

describe('Card', () => {
    it('snapshot is unchanged', () => {
        const toRender = (
            <Card>
                Hello World
            </Card>
        );

        const tree = renderer.create(toRender).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
