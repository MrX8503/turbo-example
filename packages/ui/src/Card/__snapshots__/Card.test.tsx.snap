// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card snapshot is unchanged 1`] = `
[
  <h1>
    Card Component
  </h1>,
  "Hello World",
]
`;
